test_scripts/Defects/8_0/2399__SDL_respond_UNSUPPORTED_RESOURCE_in_case_SDL_4_0_was_omited.lua

```diff
@@ -0,0 +1,41 @@
+---------------------------------------------------------------------------------------------------
```

Comment:
@VNikiforov-dev Please add this script to `test_sets/Defects/Defects_release_8_0.txt` test set